### PR TITLE
Compile figure S2

### PR DESCRIPTION
**Pull request by sjspielman**
Part of #47 
Edit - Stacked on #52 

This PR includes the compiled version of Figure S2. Here, I did do all text resizing to match. Everything is 6 pt Arial, except for the plot _axis titles_ in Panel C which are 8 pt Arial. Notably, I kept Panel C as a png, since it is _huge_ when exported as a PDF due to all the many scatterplot points. Then the compiled figure is even bigger (approaching 50 MB). The resolution with tiff wasn't much different from png either, so I kept it as png. Then in Illustrator, I replaced labels with little text boxes in Arial font at the right size.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
@sjspielman can you update this to reflect our decisions earlier in the week, e.g., remove the `.ai` file and make the fonts bigger since we seem to have room to do that?

**Comment by sjspielman** [comment]
This has been updated with larger fonts. As before, Panel C is a PNG that is placed into the illustrator; otherwise the figure file size gets too big (approaching 40 MB). Since this is a supplemental figure, that choice seems not so bad!

File: figures/compiled_figures/pdfs/figure_S2.pdf
Comment (jaclyn-taroni) [inline figures/compiled_figures/pdfs/figure_S2.pdf:1]: `figure_s2.pdf` (also applies to PNG) would better conform with the convention in `main`.

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻  Minor comment: please update the file names to be more consistent with what is in `main`. I don't need to take another look.

**Pull request merged**